WORKSPACE

```diff
@@ -76,11 +79,11 @@ http_archive(
 http_archive(
     name = "arrow",
     urls = [
-        "https://mirror.bazel.build/github.com/apache/arrow/archive/apache-arrow-0.9.0.tar.gz",
-        "https://github.com/apache/arrow/archive/apache-arrow-0.9.0.tar.gz",
+        "https://mirror.bazel.build/github.com/apache/arrow/archive/apache-arrow-0.10.0.tar.gz",
```

Comment:
@yongtang I'm going to try and update this to use Arrow version 0.11.1. I think it shouldn't affect anything in the Parquet Dataset, but I'm not totally sure, so I will verify first. What are your thoughts on this?